resources/ajuna/paseo/ajuna-paseo-raw-unsorted.json

```diff
@@ -1,16 +1,19 @@
 {
   "name": "Ajuna Paseo",
-  "id": "ajuna-ajuna-westend",
+  "id": "ajuna-paseo",
   "chainType": "Live",
-  "bootNodes": [],
+  "bootNodes": [
+    "/ip4/95.217.105.80/tcp/23804/p2p/p2p/12D3KooWFR6k9RLLRUJDhS8R4a2s7c8Tc1t2Rf7d7WGn9K2XpNsB",
+    "/ip4/135.181.240.254/tcp/23804/p2p/p2p/12D3KooWLRzxpDsfbgZPVM3719JQ1Xct5jg3En7ANJZ2ntRfYo7x"
+  ],
```

Comment:
We should add bootnodes to all our chain-specs like this. Ideally more than one.

Reply:
why is p2p mentioend two time? `/p2p/p2p/`

Reply:
this is a search-repleace error....